pyvrp/GeneticAlgorithm.py

```diff
@@ -23,13 +23,17 @@
 class GeneticAlgorithmParams:
     repair_probability: float = 0.80
     collect_statistics: bool = False
-    should_intensify: bool = True
+    intensification_probability: float = 0.15
```

Comment:
What about `intensify_probability`? That meshes with `intensify_on_best`, and is quite a bit shorter.